src/components/Global/MarkdownContent.vue

```diff
@@ -0,0 +1,25 @@
+<script setup>
```

Comment:
This component will handle any markdown string and convert it to html, then `v-html` it in to a div with the class of markdown. The benefit is that we can use markdown in the localizations, and then style the resulting html. This does make it a littler harder to use vuetify but I think it's ok for blocks of text that should be colocated anyway.